src/main/java/com/fismobile/appium/base/BaseClass.java

```diff
@@ -75,15 +73,13 @@ private WebDriver setup()
                 } catch (MalformedURLException e) {
                     e.printStackTrace();
                 }
-                driver.get("http://saucelabs.com/test/guinea-pig");
+                driver.get(FileReaderManager.getInstance().getConfigReader().getAppUrl());
                 logger.debug("Appium driver started. Thread ID = " + Thread.currentThread().getId());
                 break;
             default:
                 System.out.println("Incorrect Platform...");
                 break;
         }
-        return driver;
-
     }
 
     private DesiredCapabilities setupDevice(String configurationFile) {
```

Comment:
In line 90 you cand perform the object casting you're doing in line 96
Both catch clauses from lines 91 and 93 could be in one single clause (https://stackoverflow.com/a/3495968)

Reply:
nice!